src/Core__Array.res

```diff
@@ -11,6 +13,28 @@ external fromArrayLikeWithMap: (Js.Array2.array_like<'a>, 'a => 'b) => array<'a>
 @val external fromIterator: Core__Iterator.t<'a> => array<'a> = "Array.from"
 @val external fromIteratorWithMap: (Core__Iterator.t<'a>, 'a => 'c) => array<'a> = "Array.from"
 
+let make = (len, x) =>
```

Comment:
Maybe a named argument `~length` should be used here.

Reply:
I agree, I think that might make it clearer.

Reply:
Should this apply to `init` too then? Or is it just because both arguments could be of the same type in this case?

Reply:
I think it should apply to both functions.

Reply:
Added in https://github.com/rescript-association/rescript-core/pull/49/commits/cef5768e050ef45d4b2f3e31087d1db94ae11ebb